### ISSUE TITLE
Telegram fails to send rpc message

### ISSUE THREAD

**Issue by nuaimat** [freqtrade/freqtrade#8070]
## Describe your environment
* Operating system: Docker running on Linux host
* Python Version: 3.10.7
* CCXT version: 2.4.60
* Freqtrade Version: stable (docker) - freqtradeorg/freqtrade:stable 

## Describe the problem:
telegram fails to send rpc message, looking at the logs i see:

```
2023-01-28 08:42:39,215 - freqtrade.persistence.trade_model - INFO - Marking Trade(id=1062, pair=RNDR/USDT, amount=318.53210000, is_short=False, leverage=1.0, open_rate=0.99730000, open_since=closed) as closed as the trade is fulfilled and found no open orders for it.
2023-01-28 08:42:39,224 - freqtrade.rpc.rpc_manager - INFO - Sending rpc message: {'type': exit_fill, 'trade_id': 1062, 'exchange': 'Kucoin', 'pair': 'RNDR/USDT', 'leverage': 1.0, 'direction': 'Long', 'gain': 'profit', 'limit': 1.0404, 'order_rate': 1.0404, 'order_type': '', 'amount': 318.5321, 'open_rate': 0.9973, 'close_rate': 1.0404, 'current_rate': None, 'profit_amount': 12.43058779, 'profit_ratio': 0.03905215, 'buy_tag': '20 ', 'enter_tag': '20 ', 'sell_reason': 'sell_profit_maximizer_01_m ( 20 )', 'exit_reason': 'sell_profit_maximizer_01_m ( 20 )', 'open_date': datetime.datetime(2023, 1, 28, 7, 37, 8, 878598), 'close_date': datetime.datetime(2023, 1, 28, 8, 42, 39, 215133), 'stake_amount': 317.67206333, 'stake_currency': 'USDT', 'fiat_currency': 'USD', 'sub_trade': False, 'cumulative_profit': 0.0}
2023-01-28 08:43:09,060 - freqtrade.worker - INFO - Bot heartbeat. PID=1, version='docker-2023.1.dev-020dc3c6, strategy_version: v11.3.100', state='RUNNING'
2023-01-28 08:43:28,887 - freqtrade.rpc.telegram - ERROR - Exception occurred within Telegram module
Traceback (most recent call last):
  File "/freqtrade/freqtrade/rpc/telegram.py", line 81, in wrapper
    return command_handler(self, *args, **kwargs)
  File "/freqtrade/freqtrade/rpc/telegram.py", line 1120, in _trades
    [[arrow.get(trade['close_date']).humanize(),
  File "/freqtrade/freqtrade/rpc/telegram.py", line 1122, in <listcomp>
    f"{(trade['close_profit']):.2%} ({trade['close_profit_abs']})"]
TypeError: unsupported format string passed to NoneType.__format__
2023-01-28 08:43:58,292 - freqtrade.rpc.telegram - ERROR - Exception occurred within Telegram module
Traceback (most recent call last):
  File "/freqtrade/freqtrade/rpc/telegram.py", line 81, in wrapper
    return command_handler(self, *args, **kwargs)
  File "/freqtrade/freqtrade/rpc/telegram.py", line 1120, in _trades
    [[arrow.get(trade['close_date']).humanize(),
  File "/freqtrade/freqtrade/rpc/telegram.py", line 1122, in <listcomp>
    f"{(trade['close_profit']):.2%} ({trade['close_profit_abs']})"]
TypeError: unsupported format string passed to NoneType.__format__
```

### Steps to reproduce:
sometimes, when an order is closed on kucoin, i don't get telegram message, when i look at the logs i see the above error log
 
### Observed Results:
* What happened?
  no message was sent to telegram
* What did you expect to happen?
  message to be sent to telegram

**Comment by xmatthias** [comment]
can you share a bit more logs from before this event? (a few lines).

this is only showing the symptom - not what's causing this.

**Comment by xmatthias** [comment]
Nevermind - traced this down to ccxt ... https://github.com/ccxt/ccxt/issues/16669 (as mentioned above - the log shows the symptom - not the cause).

Please note: you specified the docker version as ":stable" - but your logs indicate that you're running the develop version - with a commit from 2 days ago.
The ccxt version will also not be matching - assuming  you use the develop image, it'll be something > 2.6.56.

You should be able to have this "back" by really using the stable image - or reverting your installation to the latest stable release (in non-docker cases).

**Comment by xmatthias** [comment]
Fixed/closed with fee7b792e1c5641cbdc7021ab66485986d5c5ff3

**Comment by nuaimat** [comment]
you're absolutely right, i was jumping between stable/develop branches trying to figure out the issue. thanks for fixing this.
